webview/src/shared/variables.scss

```diff
@@ -10,6 +10,7 @@ $deps-color: var(--vscode-dvc-deps);
 $changed-color: var(--vscode-dvc-workspaceChanged);
 $error-color: var(--vscode-errorForeground);
 
+$experiments-view-bg-color: var(--vscode-menu-background);
 $row-bg-color: $bg-color;
 $header-fg-color: $fg-color;
 $row-bg-alt-color: var(--vscode-sideBar-background);
```

Comment:
[Q] Unrelated to this PR but can we change the name of this? It is only use for on-hover in the table.